### ISSUE TITLE
[BUG] Improve/fix handling for incorrect epicbox address

### ISSUE THREAD

**Issue by blacktyger** [EpicCash/epic-wallet#88]
During some tests I've discovered that invalid epicbox address is not handled as expected, it is related to both: `cli` and `owner_api`.

- Using CLI API command:

```bash
./epic-wallet send -m epicbox -d incorrect_address@epicbox.epic.tech 1
```
wallet will panic with RUST traceback error:

```2023-07-09 22:23:37.587 ERROR                                                                                                                                     
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Error { inner: LibWallet Error: Invalid base58 checksum }': 
  0: epic_util::logger::send_panic_to_log::{{closure}}
   1: std::panicking::rust_panic_with_hook                                                                                                                        
             at /rustc/7737e0b5c4103216d6fd8cf941b7ab9bdbaace7c/library/std/src/panicking.rs:702:17                                                               
   2: std::panicking::begin_panic_handler::{{closure}}                                                        
             at /rustc/7737e0b5c4103216d6fd8cf941b7ab9bdbaace7c/library/std/src/panicking.rs:588:13           
   3: std::sys_common::backtrace::__rust_end_short_backtrace                                                  
             at /rustc/7737e0b5c4103216d6fd8cf941b7ab9bdbaace7c/library/std/src/sys_common/backtrace.rs:138:18
   4: rust_begin_unwind                                                                                       
             at /rustc/7737e0b5c4103216d6fd8cf941b7ab9bdbaace7c/library/std/src/panicking.rs:584:5
   5: core::panicking::panic_fmt
             at /rustc/7737e0b5c4103216d6fd8cf941b7ab9bdbaace7c/library/core/src/panicking.rs:143:14
   6: core::result::unwrap_failed
             at /rustc/7737e0b5c4103216d6fd8cf941b7ab9bdbaace7c/library/core/src/result.rs:1749:5
   7: epic_wallet_impls::adapters::epicbox::EpicboxChannel::send
   8: epic_wallet_controller::controller::owner_single_use
   9: epic_wallet_controller::command::send
  10: epic_wallet::cmd::wallet_args::wallet_command
  11: epic_wallet::cmd::wallet::wallet_command
  12: epic_wallet::real_main
  13: epic_wallet::main
  14: std::sys_common::backtrace::__rust_begin_short_backtrace
  15: std::rt::lang_start::{{closure}}
  16: core::ops::function::impls::<impl core::ops::function::FnOnce<A> for &F>::call_once
             at /rustc/7737e0b5c4103216d6fd8cf941b7ab9bdbaace7c/library/core/src/ops/function.rs:259:13
      std::panicking::try::do_call
             at /rustc/7737e0b5c4103216d6fd8cf941b7ab9bdbaace7c/library/std/src/panicking.rs:492:40
      std::panicking::try
             at /rustc/7737e0b5c4103216d6fd8cf941b7ab9bdbaace7c/library/std/src/panicking.rs:456:19
      std::panic::catch_unwind
             at /rustc/7737e0b5c4103216d6fd8cf941b7ab9bdbaace7c/library/std/src/panic.rs:137:14
      std::rt::lang_start_internal::{{closure}}
             at /rustc/7737e0b5c4103216d6fd8cf941b7ab9bdbaace7c/library/std/src/rt.rs:128:48
      std::panicking::try::do_call
             at /rustc/7737e0b5c4103216d6fd8cf941b7ab9bdbaace7c/library/std/src/panicking.rs:492:40
      std::panicking::try
             at /rustc/7737e0b5c4103216d6fd8cf941b7ab9bdbaace7c/library/std/src/panicking.rs:456:19
      std::panic::catch_unwind
             at /rustc/7737e0b5c4103216d6fd8cf941b7ab9bdbaace7c/library/std/src/panic.rs:137:14
      std::rt::lang_start_internal
             at /rustc/7737e0b5c4103216d6fd8cf941b7ab9bdbaace7c/library/std/src/rt.rs:128:20
  17: main
  18: <unknown>
  19: __libc_start_main
  20: _start

Thread 'main' panicked with message:
"called `Result::unwrap()` on an `Err` value: Error { inner: LibWallet Error: Invalid base58 checksum }"
```

- Using `owner_api` to send transaction with `init_send_tx` endpoint I'm getting very generic and misleading error:

```Connection error, is wallet api running under: http://127.0.0.1:3420/v3/owner?```

---

It would be great to have that handled gracefully for cli and more specific error log for `owner_api` case. It is worth to note that epicbox addresses **are case sensitive**, at least for the base58 public key part.

**Comment by who-biz** [comment]
Will add this into the prospective Epicbox improvements, thanks for report

**Comment by blacktyger** [comment]
fixed with #91
